Std/Tactic/TryThis.lean

```diff
@@ -75,65 +263,140 @@ partial def replaceMVarsByUnderscores [Monad m] [MonadQuotation m]
 def delabToRefinableSyntax (e : Expr) : TermElabM Term :=
   return ⟨← replaceMVarsByUnderscores (← delab e)⟩
 
+/-- Delaborate `e` into a suggestion suitable for use in `refine`. -/
```

Comment:
```suggestion
/-- Delaborate `e` into a suggestion suitable for use in `refine`. -/
```
"in `refine`" seems wrong here?

Reply:
You're right—this is vestigial copypasta from the docstring above it that I forgot to review. :) I'll figure out an appropriate rephrasing tomorrow.

Reply:
Oops, this one fell through the cracks. I changed the preposition to `by` in 830fc8c, but I've stared at it too long for me to tell if that sounds right...